frontend/core/blocks/watermarking/computations.py

```diff
@@ -0,0 +1,100 @@
+import os
+from PIL import Image, ImageDraw, ImageFont
+
+def add_watermark(input_image_path, output_image_path, watermark_text, position, transparency=128):
+    """
+    Adds a watermark to an image.
+
+    Args:
+        input_image_path (str): Path to the input image.
+        output_image_path (str): Path to save the watermarked image.
+        watermark_text (str): The text of the watermark.
+        position (str): Position of the watermark ('bottom_right', 'bottom_left', 'top_right', 'top_left', 'center').
+        transparency (int): Transparency level of the watermark (0-255).
+
+    Returns:
+        Image: The watermarked image.
+    """
+    # Open the original image
+    original = Image.open(input_image_path).convert("RGBA")
+
+    # Make the image editable
+    txt = Image.new('RGBA', original.size, (255, 255, 255, 0))
+
+    # Choose a font and size
+    font = ImageFont.truetype("arial.ttf", 500)
```

Comment:
Do we need the rest of the .ttf files as well?

Reply:
yes those the formats that comes in bundle with arial.ttf that is why i kept those. It gives an error if one of the file is missing